### ISSUE TITLE
Add resource to test AWS VPC Endpoints and their policies

### ISSUE THREAD

**Issue by JQUINONES82** [inspec/inspec-aws#21]
### 🎛 Description

feature request

Would like to monitor and detect S3 VPC Endpoint policies for changes or certain configuration settings.

For example, from a security context, being able to limit outbound access to S3 buckets via an endpoint to a specific set of buckets.

This is done to limit egress points out of the VPC for data exfiltration? 

Inspec can be used to detect a change or state of the policy.

**Comment by skpaterson** [comment]
We'll look to add a resource complementing https://www.terraform.io/docs/providers/aws/r/vpc_endpoint.html

**Comment by omer-chefio** [comment]
@brasskazoo Please see the docs for the `aws_vpc` resource [here](https://github.com/inspec/inspec-aws/blob/master/docs/resources/aws_vpc.md).

This should hopefully resolve your issue.

**Comment by anupammajhi** [comment]
> @brasskazoo Please see the docs for the `aws_vpc` resource [here](https://github.com/inspec/inspec-aws/blob/master/docs/resources/aws_vpc.md).
> 
> This should hopefully resolve your issue.

I don't think this solves the issue. The feature request is for "VPC Endpoints" and not "VPC" per my understanding.

Can someone correct me if I am missing something here.

Otherwise I would request to reopen this issue.

**Comment by anupammajhi** [comment]
@omerdemirok I believe this can be closed now.
